src/components/panels/AlertPanel.vue

```diff
@@ -1,62 +1,51 @@
 <template>
-  <b-col md="6">
-    <h3 class="text-center">Alerts</h3>
-    <b-list-group>
-      <b-list-group-item :style="styleObject" v-for="alert in getAlerts" :key="alert.id" class="list-group-item-borderless">
```

Comment:
why did you change it from using data to using a prop?

Reply:
its the way vue does

Reply:
um, no? cause I already had it as data.... _in vue_

Reply:
why  do a  round over computed prop  instead of     sending it directly ?

Reply:
"a round over"?

the previous way, it's fetched from the state by the component. your way, it's passed to the component, with no validation that it's the right object.

Reply:
my point is why did you change the existing pattern without discussion?